lib/audited/auditor.rb

```diff
@@ -219,11 +219,30 @@ def write_audit(attrs)
         run_callbacks(:audit)  { audits.create(attrs) } if auditing_enabled
       end
 
+      def presence_of_audit_comment
+        case
+        when !auditing_enabled
+          return true
+        when audit_comment.present?
+          return true
+        when audited_options[:on].present? && audited_options[:on].exclude?(:create) && self.new_record?
```

Comment:
no need to check `audited_options[:on].present?` here and below, the `normalize_audited_options` function will make sure it's there and contains at least one option.

Reply:
Ah, good call. I wasn't aware of the `normalize_audited_options` function. Thanks!